pkg/util/mount/mount.go

```diff
@@ -25,7 +25,7 @@ import (
 	"k8s.io/kubernetes/pkg/util/exec"
 )
 
-type Interface interface {
+type Mounter interface {
```

Comment:
I would argue don't double-embed.  Keep field names.  But I don't like the system object at all, as I am sure you saw by now.